node/src/components/sync_leaper/tests.rs

```diff
@@ -0,0 +1,374 @@
+use std::{collections::BTreeSet, sync::Arc};
+
+use casper_types::testing::TestRng;
+use prometheus::Registry;
+
+use crate::{
+    components::{
+        fetcher::{self, FetchResult, FetchedData},
+        sync_leaper::{LeapState, PeerState, RegisterLeapAttemptOutcome},
+    },
+    types::{Block, BlockHash, Chainspec, NodeId, SyncLeap, SyncLeapIdentifier},
+};
+
+use super::{Error, SyncLeaper};
+
+pub(crate) fn make_default_sync_leap(rng: &mut TestRng) -> SyncLeap {
```

Comment:
I don't think `make_default_sync_leap` is a suitable name here because it's based on a random block, which has no defaults.

Reply:
Renamed to `make_test_sync_leap()` in afe2163ce3d4f7bbbcb62208da4dec65c5da75b8